com.unity.testing.visualeffectgraph/PerformanceTests/Runtime/VFXPerformanceRuntimeTests.cs

```diff
@@ -0,0 +1,214 @@
+        {
+            yield break;
+        }
+
+        static IEnumerable<string> allMarkerName
+        {
+            get
+            {
+                yield return "VFX.MeshSystem.Render";
+                yield return "VFX.ParticleSystem.CopySpawnEventAttribute";
+                yield return "VFX.ParticleSystem.CopyEventListCountCommand";
+                yield return "VFX.ParticleSystem.CopyCount";
+                yield return "VFX.ParticleSystem.ResetCount";
+                yield return "VFX.ParticleSystem.CopyIndirectCount";
+                yield return "VFX.ParticleSystem.UpdateMaterial";
+                yield return "VFX.ParticleSystem.Init";
+                yield return "VFX.ParticleSystem.Update";
+                yield return "VFX.ParticleSystem.PerStripUpdate";
+                yield return "VFX.ParticleSystem.RenderPoint";
+                yield return "VFX.ParticleSystem.RenderPointIndirect";
+                yield return "VFX.ParticleSystem.RenderLine";
+                yield return "VFX.ParticleSystem.RenderLineIndirect";
+                yield return "VFX.ParticleSystem.RenderTriangle";
+                yield return "VFX.ParticleSystem.RenderTriangleIndirect";
+                yield return "VFX.ParticleSystem.RenderQuad";
+                yield return "VFX.ParticleSystem.RenderQuadIndirect";
+                yield return "VFX.ParticleSystem.RenderOctagon";
+                yield return "VFX.ParticleSystem.RenderOctagonIndirect";
+                yield return "VFX.ParticleSystem.RenderHexahedron";
+                yield return "VFX.ParticleSystem.RenderHexahedronIndirect";
+                yield return "VFX.ParticleSystem.RenderMesh";
+                yield return "VFX.ParticleSystem.RenderMeshIndirect";
+                yield return "VFX.Update";
+                yield return "VFX.PrepareCamera";
+                yield return "VFX.ProcessCamera";
+                yield return "VFX.FillIndirectRenderArgs";
+                yield return "VFX.CopyBuffer";
+                yield return "VFX.InitializeDeadListBuffer";
+                yield return "VFX.ZeroInitializeBuffer";
+                yield return "VFX.SortBuffer";
+                yield return "VFX.NotifyModifiedAsset";
+                yield return "VFX.NotifyDeletedAsset";
+                yield return "VFX.DefaultCommandBuffer";
+                yield return "VFX.RegisterGizmos";
+                yield return "VFXEditor.VisualEffectImporter.GenerateAssetData";
+                yield return "VFXEditor.VisualEffectImporter.GenerateAssetDataOneShader";
+                yield return "VFXEditor.VisualEffectResource.GetResourceAtPath";
+                yield return "VFXEditor.VisualEffectResource.GetResourceAtPath_Depoint";
+            }
+        }
+
+        public static IEnumerator Load_And_Prepare(GraphicsTestCase testCase)
+        {
+            UnityEngine.SceneManagement.SceneManager.LoadScene(testCase.ScenePath);
+            yield return null;
```

Comment:
Replace `yield return null;` by `yield return new WaitForEndOfFrame()` to always process a frame.